docs/how-to/hip_programming_guide.rst

```diff
@@ -0,0 +1,32 @@
+.. meta::
+    :description: HIP programming guide
+    :keywords: HIP, heterogeneous programming, ROCm API, AMD GPU programming
+
+.. _hip-programming-guide:
+
+********************************************************************************
+HIP programming guide
+********************************************************************************
```

Comment:
This currently is not specifically a "HIP" programming guide, and also doesn't link to the HIP programming guide, but just the index of the HIP documentation.

Reply:
Updated?